ts/blueprints/ember-cli-typescript/index.js

```diff
@@ -3,16 +3,36 @@
 const fs = require('fs');
 const path = require('path');
 
-const APP_DECLARATIONS = `
-import Ember from 'ember';
+const APP_DECLARATIONS = `import Ember from 'ember';
```

Comment:
unrelated fix to avoid an unnecessary newline before the `types/my-app/index.d.ts` imports